packages/trader/src/App/Components/Elements/ContractAudit/contract-audit.tsx

```diff
@@ -18,7 +18,9 @@ type TContractAudit = Partial<
     duration_unit: string;
     exit_spot: string | undefined;
     is_dark_theme: boolean;
+    is_history_tab_active: boolean;
     is_open: boolean;
+    current_lang_to_request_history: boolean | string;
```

Comment:
Type `boolean | string` is a bit unexpected. Sorry, but could you please explain when it can be boolean? 
Please, ignore if irrelevant

Reply:
Ahh, found this `current_lang_to_request_history={!is_language_changing && current_language}`

Reply:
When update history should not be requested (current language has not yet finished changing), it returns `false`, and we don't request it as per condition inside `useEffect`.
When update history should be requested, it returns that latest selected (current) language ('EN'), which should be a `string` in order to trigger `useEffect` when its value changes to another language ('FR').

Reply:
i'll consider updating it to `current_lang_to_request_history?: string;`, might still work the same way :)